cloudferrylib/os/compute/nova_compute.py

```diff
@@ -465,6 +467,16 @@ def _deploy_resources(self, info, **kwargs):
 
         return new_info
 
+    def check_failed_instances_list(self):
```

Comment:
This should be moved to the latest stages of VM migration, perhaps new task/action.